Src/beebwin.h

```diff
@@ -392,6 +447,7 @@ class BeebWin {
 	bool		m_EmuPaused;
 	bool		m_StartPaused;
 	bool		m_WasPaused;
+	bool		m_J2KWasEnabled;
```

Comment:
I'm not sure I follow what this variable is being used for

Reply:
Joystick to keyboard mapping is temporarily enabled for displaying mapping dialog. This variable keeps previous state, to restore when the dialog is closed.